frontend/src/app/features/personal-cabinet/components/create-folder-modal/create-folder-modal.component.scss

```
@@ -0,0 +1,179 @@
+    &:hover:not(:disabled) {
+      background: var(--accent-hover);
+    }
+
+    &:disabled {
+      opacity: 0.5;
+      cursor: not-allowed;
+    }
+  }
+}
+
+@keyframes slideUp {
```

Stylelint (17.9.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)